src/sentry/testutils/factories.py

```diff
@@ -274,6 +274,11 @@ def create_environment(project, **kwargs):
         env.add_project(project, is_hidden=kwargs.get("is_hidden"))
         return env
 
+    @staticmethod
+    def create_organization_environment(organization, **kwargs):
+        name = kwargs.get("name", petname.Generate(3, " ", letters=10)[:64])
+        return Environment.objects.create(organization_id=organization.id, name=name)
```

Comment:
Environments to projects is a many-to-many relationship, but I added this because the existing testutils only support creating environments for a single project. Am I mistaken?

Reply:
You could use `environment.add_project()` to add additional projects to environments created with `create_environment`.

Reply:
I tried that and I want to say it didn't work for some reason. But I'll give it another pass now that the surroundings are more stable.

Reply:
I've remembered the problem. It's not exactly setting up the many-to-many relationship that doesn't work; I'm still able to do after the fact with `.add_project`. But `create_organization` is hard-coded to start with at least one project and to use it to set the `organization_id` field.

That is, calling `self.create_environment` with no project defaults to creating an association with `self.project` and then sets the new environment's `organization_id` to `self.project.organization_id`. There is no way to pass an organization to `self.create_environment` to set its ID short of wrapping it in a project, which I can't do here because of the chicken-and-egg problem between projects and environments.

However, in 7361dd6, I think I have an adequate workaround. It's a little awkward, but probably less so than having a second, special "create environment" method.